Sample Solutions/Data Lake/MGDC/src/JsonToCDM_CSV_DatabricksNotebook.ipynb

```diff
@@ -0,0 +1 @@
,"transient":null}],"execution_count":0}],"metadata":{"application/vnd.databricks.v1+notebook":{"notebookName":"jsontocdmModified_V2","dashboards":[],"notebookMetadata":{"pythonIndentUnit":4},"language":"python","widgets":{"RunId":{"nuid":"7f3d3e1c-e853-41c9-9eb7-1481928e19e3","currentValue":"","widgetInfo":{"widgetType":"text","name":"RunId","defaultValue":"","label":"","options":{"widgetType":"text","validationRegex":null}}},"TenantId":{"nuid":"273b9632-63d7-4f5b-8be9-33f17e28411b","currentValue":"","widgetInfo":{"widgetType":"text","name":"TenantId","defaultValue":"","label":"","options":{"widgetType":"text","validationRegex":null}}},"EventDatasetFolder":{"nuid":"55d13d1a-3410-48a1-b071-efc1d9f6688c","currentValue":"","widgetInfo":{"widgetType":"text","name":"EventDatasetFolder","defaultValue":"","label":"","options":{"widgetType":"text","validationRegex":null}}},"MessageDatasetFolder":{"nuid":"3d36cc48-5e21-47bf-a121-d8ce0cabc605","currentValue":"","widgetInfo":{"widgetType":"text","name":"MessageDatasetFolder","defaultValue":"","label":"","options":{"widgetType":"text","validationRegex":null}}},"UserDatasetFolder":{"nuid":"af3670ec-ca20-467e-9ff9-bd9d31557cb1","currentValue":"","widgetInfo":{"widgetType":"text","name":"UserDatasetFolder","defaultValue":"","label":"","options":{"widgetType":"text","validationRegex":null}}},"AppId":{"nuid":"665a1ca0-b29b-4d02-ac9d-8f8390c59680","currentValue":"","widgetInfo":{"widgetType":"text","name":"AppId","defaultValue":"","label":"","options":{"widgetType":"text","validationRegex":null}}},"CdmModelName":{"nuid":"06100a2d-85b3-4c94-b621-5ac7b55e7970","currentValue":"","widgetInfo":{"widgetType":"text","name":"CdmModelName","defaultValue":"","label":"","options":{"widgetType":"text","validationRegex":null}}},"AppKey":{"nuid":"69633631-5018-4e20-a956-ef1348ba029e","currentValue":"","widgetInfo":{"widgetType":"text","name":"AppKey","defaultValue":"","label":"","options":{"widgetType":"text","validationRegex":null}}},"OfficeDataFileSystem":{"nuid":"f59e69a8-ab6e-48f9-bbe3-a415a3b83fc1","currentValue":"","widgetInfo":{"widgetType":"text","name":"OfficeDataFileSystem","defaultValue":"","label":"","options":{"widgetType":"text","validationRegex":null}}},"CdmDataFileSystem":{"nuid":"f37447a9-ffc5-47cd-b306-bb9e925ff733","currentValue":"","widgetInfo":{"widgetType":"text","name":"CdmDataFileSystem","defaultValue":"","label":"","options":{"widgetType":"text","validationRegex":null}}},"ManagerDatasetFolder":{"nuid":"497e4d96-85bc-45e4-a086-79ec2c4a95ef","currentValue":"","widgetInfo":{"widgetType":"text","name":"ManagerDatasetFolder","defaultValue":"","label":"","options":{"widgetType":"text","validationRegex":null}}},"StorageAccountName":{"nuid":"58a94392-7eaa-4dc6-8bcb-7ecd4c080b1e","currentValue":"","widgetInfo":{"widgetType":"text","name":"StorageAccountName","defaultValue":"","label":"","options":{"widgetType":"text","validationRegex":null}}}},"notebookOrigID":2351832459914103}},"nbformat":4,"nbformat_minor":0}
```

Comment:
line breaks looks to be a challenge for this in the code review visualizations

same question around the contents here.

Also, question if this is something that was from another public repo, if so, why not use that one?

Reply:
Feature definition (from MGDC to the new format) is from an MGDC public solution but many things has changed like HDInsights to Synapse, New CDM format, CSV export etc
Will reference the original pipeline in the README

Reply:
Not sure why line breaks is an issue. Maybe something about how Databricks notebooks are exported. Will try to export again

Reply:
still looks odd but, i'd guess based on the 2 files, that the reason is the synapse one has \r\n for line breaks and the databricks one has \n

Reply:
Maybe. but don't want to modify the file because they should be able to download and upload the same .ipnyb file into a Databricks instance and this is what Databricks has exported format wise

Reply:
It does make code reviews more difficult if there are changes. if it isn't something you feel comfortable changing it can be left.